lib/Partitioner/Partitioner.cpp

```diff
@@ -62,12 +69,12 @@ void Partitioner::dumpDAG(llvm::StringRef dotFilename) const {
     for (size_t i = 0; i < node->children.size(); i++) {
       auto child = node->children[i];
       DescriptionBuilder db(child->name.c_str());
-      db.addParam("BackendKind", backendName_.at(child->backendKind));
+      const std::string &backendName = backendMap_.at(child->backendKind).name;
+      db.addParam("BackendKind", backendName);
       myfile << child->name << " [ label = \"" << escapeDottyString(db) << "\"";
```

Comment:
Please use escapeDottyString on all names/dynamic strings. I've seen some issues if we don't do it.